service/src/main/resources/HTMLTemplate.html

```diff
@@ -206,20 +206,46 @@ <h3 class="heading heading--two">Orders</h3>
       <dd class="data-list__definition">
         {% if case_data.orders is not empty and case_data.orders.orderType is not empty %}
           {% for type in case_data.orders.orderType %}
-            <span class="u-block">{{ type }}</span>
+            {{ type | ordermapping }}
```

Comment:
Please replace `toLowerCase` in `DocumentGeneratorConfiguration#buildEngine` method with something that just lower case first letter only. This way we will have camel cased filter names.

Having `{{ order | emergencyProtectionOrderMapping }}` is much more readable.